From 07c66c7857ae9f557ebcdd5394e2a85d12d3d04a Mon Sep 17 00:00:00 2001
From: Ben Rudolph <brudolph@dimagi.com>
Date: Tue, 3 Jan 2017 17:35:21 +0200
Subject: [PATCH] Location Manager utils

---
 corehq/apps/locations/models.py | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/corehq/apps/locations/models.py b/corehq/apps/locations/models.py
index 56fa535..fdff85e 100644
--- a/corehq/apps/locations/models.py
+++ b/corehq/apps/locations/models.py
@@ -341,6 +341,19 @@ class LocationManager(LocationQueriesMixin, TreeManager):
         direct_matches = self.filter_by_user_input(domain, user_input)
         return self.get_queryset_descendants(direct_matches, include_self=True)
 
+    def get_locations_and_children(self, location_ids):
+        """
+        Takes a set of location ids and returns a django queryset of those
+        locations and their children.
+        """
+        return SQLLocation.objects.get_queryset_descendants(
+            SQLLocation.objects.filter(location_id__in=location_ids),
+            include_self=True
+        )
+
+    def get_locations_and_children_ids(self, location_ids):
+        return list(self.get_locations_and_children(location_ids).location_ids())
+
 
 class OnlyUnarchivedLocationManager(LocationManager):
 
-- 
2.2.1

